### ISSUE TITLE
[ A11Y] Missing heading tag in text section

### ISSUE THREAD

**Issue by Devika008** [pkp/pkp-lib#9346]
## Issue Description
// What is the nature of the issue? How does it impact users?

In the introduction section of "Making a New Submission", the heading "Before you begin" above the introduction text is missing the heading level mark-up: H2. 

## How to reproduce / Current behavior
// Step-by-step instructions explaining how to reproduce the problem. The environment required, explicit identification of the component in question

Log in as an editor/author

1. Using the tab key on the keyboard, start to press it to move through focusable elements
2. Try to reach the "New Submission" button only using the TAB key.
3. You will encounter the problem on the first page when you click the "New Submission" button

## What application are you using?
 OJS, OMP or OPS version X.X.X
 
 OJS 3.4 

## Stack used 

- Device: laptop
- OS: MacOS (Monterey)
- Browser: Chrome
- Version: 117

## Testing method

- Manual (keyboard)
- Automated test: Accessibility Insights

## Action
// Describe action that can be taken to remediate the issue. 

Replace the span tag by Heading level 2 tag, i.e.; H2
When we do this, we will meet the WCAG Criteria of [2.4.6 Headings and Labels - AA](https://www.w3.org/WAI/WCAG21/Understanding/headings-and-labels.html)

## Additional information
// Please add any screenshots, code snippets, or other information we can use to investigate this issue.

![image](https://github.com/pkp/pkp-lib/assets/112014397/74c0c579-344c-4811-b7e9-5918a0efed51)
